cmd/common.go

```diff
@@ -614,3 +614,30 @@ func exportCodesignFiles(tool Tool, archivePath, outputDirPath string) error {
 
 	return nil
 }
+
+func trimProjectpath(projpth string) string {
+	if len(projpth) == 0 {
+		return projpth
+	}
+	characterSetToRemove := []string{"\n", `'`, `"`}
+	for {
```

Comment:
please don't use infinite loop if possible - include the condition directly in the `for (CONDITION)` part

Reply:
for example:
```
for _, character := range []string{"\n", `'`, `"`} {
...
}
```

Here you can find more info about range: https://gobyexample.com/range